tools/PI/DevHome.PI/Controls/ExpandedViewControl.xaml

```diff
@@ -30,22 +30,32 @@
                     <RowDefinition Height="Auto"/>
                 </Grid.RowDefinitions>
 
-                <ListView
-                x:Name="NavLinksList" SelectionMode="Single" VerticalAlignment="Stretch" ItemsSource="{x:Bind viewModel.Links}"
-                 SelectedIndex="{x:Bind viewModel.SelectedNavLinkIndex, Mode=TwoWay}" SelectionChanged="{x:Bind viewModel.Navigate}">
-                    <ListView.ItemTemplate>
-                        <DataTemplate>
-                            <StackPanel Orientation="Horizontal">
-                                <TextBlock Text="{Binding IconText}" Style="{x:Null}"
-                                FontFamily="Segoe Fluent Icons" FontSize="12"
-                                HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                                <TextBlock Text="{Binding ContentText}"
-                                Margin="12,0,0,0" HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                            </StackPanel>
-                        </DataTemplate>
-                    </ListView.ItemTemplate>
-                </ListView>
-
+                <StackPanel Orientation="Vertical">
+                    <ListView
+                    x:Name="NavLinksList" SelectionMode="Single" VerticalAlignment="Stretch" ItemsSource="{x:Bind viewModel.Links}"
+                     SelectedIndex="{x:Bind viewModel.SelectedNavLinkIndex, Mode=TwoWay}" SelectionChanged="{x:Bind viewModel.Navigate}">
+                        <ListView.ItemTemplate>
+                            <DataTemplate>
+                                <StackPanel Orientation="Horizontal">
+                                    <TextBlock Text="{Binding IconText}" Style="{x:Null}"
+                                    FontFamily="Segoe Fluent Icons" FontSize="12"
+                                    HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                                    <TextBlock Text="{Binding ContentText}"
+                                    Margin="12,0,0,0" HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                                </StackPanel>
+                            </DataTemplate>
+                        </ListView.ItemTemplate>
+                    </ListView>
+                    <Rectangle x:Name="Separator"

```

Comment:
Can `controls:PISeperator` be used here instead?

Reply:
I went back and forth on this one. The PISeparator has a specific width/height for separating the icons in the bar. This is used for something slightly different, and has a longer width. If I need to use this multiple times, I might just create a new separator control.